ci/scripts/python_test.sh

```diff
@@ -29,4 +29,4 @@ export LD_LIBRARY_PATH=${ARROW_HOME}/lib:${LD_LIBRARY_PATH}
 # Enable some checks inside Python itself
 export PYTHONDEVMODE=1
 
-pytest -r s --pyargs pyarrow
+pytest -r s -vvv --pyargs pyarrow
```

Comment:
Revert this?

Reply:
I think the verbosity is more helpful when hunting down a failed test, but we can discuss that in a follow up